src/environments/environment.ts

```diff
@@ -5,5 +5,5 @@
 
 export const environment = {
 	production:    false,
-	API_BASE_PATH: 'http://127.0.0.1:8080/seed/v1'
```

Comment:
The port cannot be changed due to 8080 is the default for the docker image

Reply:
Ok, done